packages/firebase/index.d.ts

```diff
@@ -7080,13 +7080,73 @@ declare namespace firebase.firestore {
    *
    *   <ul>
    *     <li><code>debug</code> for the most verbose logging level, primarily for
-   *     dubugging.</li>
+   *     debugging.</li>
    *     <li><code>error</code> to log errors only.</li>
    *     <li><code>silent</code> to turn off logging.</li>
    *   </ul>
    */
   export function setLogLevel(logLevel: LogLevel): void;
 
+  /**
+   * Converter used by `withConverter()` to transform user objects into
+   * Firestore data.
+   *
+   * Using the converter allows you to specify generic type arguments when
+   * storing and retrieving objects from Firestore.
+   *
+   * @example
+   * ```typescript
+   * class Post {
+   *   constructor(readonly title: string, readonly author: string) {}
+   *
+   *   toString(): string {
+   *     return this.title + ', by ' + this.author;
+   *   }
+   * }
+   *
+   * const postConverter = {
```

Comment:
If you make it a class, then they would have to do `withConverter(new PostConverter())` instead of `withConverter(PostConverter)` so I'm not sure it's an improvement...